airlock/views/request.py

```diff
@@ -428,8 +428,11 @@ def file_approve(request, request_id, path: str):
     except bll.FileNotFound:
         raise Http404()
 
+    urlpath = UrlPath(path)
+    group_name = urlpath.parts[0]
```

Comment:
If line 427 changed from:
```
relpath = release_request.get_request_file_from_urlpath(path).relpath
```
to:
```
request_file = release_request.get_request_file_from_urlpath
```
you can get the group name from the `request_file` instead of parsing the path.

Reply:
In fact, you could pass the entire `request_file` to `approve_file`/`reject_file` instead of `relpath` and `group_name`

Reply:
refactored, wdyt @rebkwok ?